src/controllers/expenses.controller.js

```diff
@@ -0,0 +1,86 @@
+const { expensesService } = require('../services/expense.services.js');
+const { usersService } = require('../services/user.services.js');
+
+const getExpenses = (req, res) => {
+  const { userId, categories, from, to } = req.query;
+  const categoryList = categories ? categories.split(',') : [];
+
+  const expenses = expensesService.getExpenses({
+    userId,
+    categoryList,
+    from,
+    to,
+  });
+
+  res.json(expenses);
+};
+
+const createExpense = (req, res) => {
+  const { userId, spentAt, title, amount, category, note } = req.body;
+
+  if (!userId || !spentAt || !title || !amount || !category || !note) {
+    return res.status(400).json({ message: 'Missing required parameters' });
```

Comment:
The error message 'Missing required parameters' could be more descriptive by specifying which parameters are missing. Consider listing the missing parameters in the response.